### ISSUE TITLE
transaction_add_action_

### ISSUE THREAD

**Issue by encoderlee** [uuosio/go-uuoskit#2]
'authorization' in  eos transaction is a list, the order of actor is very important, such as ONLY_BILL_FIRST_AUTHORIZER feature,  the first actor will pay for cpu, but in transaction_add_action_(), 'permissions' is dict , after json.Unmarshal, the original order of actor will not be maintained...

**Comment by learnforpractice** [comment]
Use `Action.AddPermission` to add permission if you want to keep it in order.

https://github.com/uuosio/go-uuoskit/blob/284a4fdc1d7281bccc5bc5343916e1bb54778f3d/uuoskit/action.go#L134

**Comment by encoderlee** [comment]
thanks, and how to use Action.AddPermission in [pyeoskit](https://github.com/learnforpractice/pyeoskit)?
in fact, I discovered this problem while using pyeoskit ..

**Comment by learnforpractice** [comment]
Provide a test code so I can replicate the problem you said.

**Comment by encoderlee** [comment]
```
from pyeoskit import eosapi, wallet

wallet.import_key('fuckpayforit', 'private key here')
wallet.import_key('consumer1111', 'private key here')
eosapi.set_node('https://wax.pink.gg')

def push_transaction(trx: dict):
    actions = []
    for item in trx["actions"]:
        authorization = {item["actor"]: item["permission"] for item in item["authorization"]}
        actions.append((item["account"], item["name"], item["data"], authorization))
    resp = eosapi.push_actions(actions)
    return resp


def main():
    trx = {
        "actions": [{
            "account": "boost.wax",
            "name": "noop",
            "authorization": [{
                "actor": "fuckpayforit",
                "permission": "active",
            }, {
                "actor": "consumer1111",
                "permission": "active",
            }],
            "data": {},
        }]
    }
    resp = push_transaction(trx)
    print(resp)


if __name__ == '__main__':
    main()
```
transaction record：https://wax.bloks.io/transaction/8ab6d05371f6c0e6685f5c2d56b51d09f1798ecba6ab13ab86d14e09c7c97f26


![image](https://user-images.githubusercontent.com/9878012/160855626-7fa84391-370a-4010-9bef-be74095ab4c1.png)
you can see the order of actor  is different

**Comment by learnforpractice** [comment]
Thank you! Sounds like there is a bug in the code. Give me some time to fix it.

**Comment by learnforpractice** [comment]
Fixed, please update pyeoskit to version 1.1.6 with the following command

```
python3 -m pip install -U pyeoskit
```

**Comment by encoderlee** [comment]
thanks!
